hie.yaml

```diff
@@ -0,0 +1,6 @@
+cradle:
+  cabal:
+    - path: "./src"
+      component: "lib:grpc-mqtt"
+    - path: "./test"
+      component: "test:test"
```

Comment:
Is this file necessary? It seems to work fine for me without this. HLS will create and use a default `hie.yaml` if none is present.

Reply:
My HLS fails during initialization with an error message complaining about something along the lines of `cabal --exec 'ghc --version'` without it for some reason. I figure it doesn't hurt to add and may become necessary once the `template-haskell` parts of `grpc-mqtt` get split into a separate package.